doc/releases/changelog-0.36.0.md

```diff
@@ -31,41 +31,117 @@
 
 <h4>Access an extended arsenal of quantum algorithms 🏹</h4>
 
-* The `FABLE` template is added for efficient block encoding of matrices. Users can now call FABLE to efficiently construct circuits according to a user-set approximation level. 
+* The Fast Approximate BLock-Encodings (FABLE) algorithm for embedding
+  a matrix into a quantum circuit
+  (https://arxiv.org/abs/2205.00081) is now accessible via the `qml.FABLE` 
+  template.
   [(#5107)](https://github.com/PennyLaneAI/pennylane/pull/5107)
 
-* Create the `qml.Reflection` operator, useful for amplitude amplification and its variants.
-  [(#5159)](https://github.com/PennyLaneAI/pennylane/pull/5159)
+  The usage of `qml.FABLE` is similar to `qml.BlockEncode` but provides a more
+  efficient circuit construction at the cost of a user-defined approximation 
+  level, `tol`. The number of wires that `qml.FABLE` operates on is `2*n + 1`, 
+  where `n` defines the dimension of the :math:`2^n \times 2^n` matrix that we
+  want to block-encode.
 
   ```python
-  @qml.prod
-  def generator(wires):
-        qml.Hadamard(wires=wires)
-
-  U = generator(wires=0)
-
+  A = np.array([[0.1, 0.2], [0.3, 0.4]])
   dev = qml.device('default.qubit')
+
   @qml.qnode(dev)
   def circuit():
+      qml.FABLE(A, tol=0.1, wires=range(3))
+      return qml.state()
+  ``` 
 
-        # Initialize to the state |1>
-        qml.PauliX(wires=0)
+  ```pycon
+  >>> circuit()
+  tensor([0.04929927+0.j, 0.15067163+0.j, 0.04929927+0.j, 0.15067163+0.j,
+          0.49756365+0.j, 0.47675786+0.j, 0.49756365+0.j, 0.47675786+0.j], requires_grad=True)
+  ```
 
-        # Apply the reflection
-        qml.Reflection(U)
+* Reflecting about a given quantum state is now available via `qml.Reflection`.
```

Comment:
@trbromley re: https://github.com/PennyLaneAI/pennylane/pull/5574#discussion_r1584879331

I thought you were referencing the other comment you made about `Reflection` being put above `AmplitudeAmplification` 😅. If I read the changelog top-down, `Reflection` being put first makes sense to me because it builds into / is a subroutine in `AmplitudeAmplification`

Reply:
I dropped a comment in that thread too, but my thinking is that it's nicer to see the important thing first, i.e., amplitude amplification. Then, by the way, we have support for `Reflection` too which is an important component of that.

But this doesn't block merging the PR though.

Reply:
I agree with Tom  :) `AmplitudeAmplification` is the important one here